router/functionHandler.go

```diff
@@ -168,6 +168,14 @@ func (roundTripper RetryingRoundTripper) RoundTrip(req *http.Request) (resp *htt
 
 	executingTimeout := roundTripper.funcHandler.tsRoundTripperParams.timeout
 
+	var reqBody []byte
+	if req.Body != nil {
+		reqBody, err = ioutil.ReadAll(req.Body)
```

Comment:
This puts the full request in the router's memory, which has a couple of downsides -- router memory usage increases (req body could be large), and it prevents us from supporting streaming (starting a function before the full request has come in).